cmd/chaos-dashboard/main.go

```diff
@@ -39,7 +39,7 @@ import (
 )
 
 // @title Chaos Mesh Dashboard API
-// @version 2.2
+// @version 2.5
```

Comment:
A little curious about where we configure for this `version`?

Reply:
Currently no place. I used to modify it manually.